cafebabel/templates/404.html

```diff
@@ -0,0 +1,91 @@
+{# Beware of not adding any external dependencies here! #}
+<!doctype html>
+<meta charset=utf-8>
+<title>404 error</title>
+<!-- Lightest blank gif, avoids an extra query to the server. -->
+<link rel=icon href="data:;base64,iVBORw0KGgo=">
+<style>
+  html {
+    font-size: 62.5%;
+    text-size-adjust: 100%;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-smoothing: antialiased;
+  }
+  * {
+    box-sizing: border-box;
+  }
+  @font-face {
+    font-family: 'vremena_grotesk';
```

Comment:
Do we really need a dedicated font for that page?

Reply:
it's a always the same question with graphic design: do we _really_ need this or that. The answer is no, we dont _really_ need it. But it's better, more consistant and it _really_ does matter. So what do you think?